erica/erica_legacy/request_processing/requests_controller.py

```diff
@@ -179,13 +181,13 @@ def generate_json(self, pyeric_response: PyericResponse):
         return response
 
 
-class CheckTaxNumberRequestController:
+class CheckTaxNumberRequestController(EricaRequestController):
```

Comment:
I know this is kind of ugly because we don't make any call to erica here. Should we add a new BaseRequestController or should we ignore it for now?

Reply:
I agree. It is not that nice and this was basically the reason why it was not an EricaRequestController before. However, we call Eric and we also use that if we get an request to erica. Therefore, I think It would be fine to leave it like that for now, wdyt?